app/(inferenceai)/inferenceai/rag-chatbot/[lang]/page.tsx

```diff
@@ -1,6 +1,38 @@
 import { SupportedLanguage } from "@/locales/.generated/types";
 import Home from "./home";
 
+import { Metadata } from "next";
+import { data } from "../../data";
+import {
+  ragChatbotHeroDesc,
+  ragChatbotHeroHeading
+} from "@/locales/.generated/server";
+
+export async function generateMetadata(): Promise<Metadata> {
```

Comment:
If it is static like this, you are better off using [static metadata](https://nextjs.org/docs/app/getting-started/metadata-and-og-images#static-metadata) to make it simpler.